libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -21,11 +21,14 @@ namespace Microsoft.Bot.Builder.LanguageGeneration
     /// </summary>
     public class Evaluator : LGTemplateParserBaseVisitor<object>
     {
+        /// <summary>
+        /// A symbol of a string constant "lgType".
```

Comment:
>A symbol of a string constant "lgType". [](start = 12, length = 39)

Better if you describe what this symbol is used for.

Reply:
This symbol can be modified as internal string constant.